packages/commerce-sdk-react/src/hooks/ShopperLogin/query.ts

```diff
@@ -13,42 +13,6 @@ import * as queryKeyHelpers from './queryKeyHelpers'
 
 type Client = ApiClients['shopperLogin']
 
-/**
- * Get credential quality statistics for a user.
- * @group ShopperLogin
- * @category Query
- * @parameter apiOptions - Options to pass through to `commerce-sdk-isomorphic`, with `null` accepted for unset API parameters.
- * @parameter queryOptions - TanStack Query query options, with `enabled` by default set to check that all required API parameters have been set.
- * @returns A TanStack Query query hook with data from the Shopper Login `retrieveCredQualityUserInfo` endpoint.
- * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-login?meta=retrieveCredQualityUserInfo| Salesforce Developer Center} for more information about the API endpoint.
- * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperlogin.shopperlogin-1.html#retrievecredqualityuserinfo | `commerce-sdk-isomorphic` documentation} for more information on the parameters and returned data type.
- * @see {@link https://tanstack.com/query/latest/docs/react/reference/useQuery | TanStack Query `useQuery` reference} for more information about the return value.
- */
-export const useCredQualityUserInfo = (
```

Comment:
Are we not using this ?

Reply:
We were not using it anywhere and the endpoint itself has been removed from SCAPI.
The API changes are released as well.